charts/vald/values.yaml

```diff
@@ -830,6 +830,13 @@ defaults:
       # @schema {"name": "defaults.observability.trace.enabled", "type": "boolean"}
       # defaults.observability.trace.enabled -- trace enabled
       enabled: false
+  # @schema {"name": "defaults.networkPolicy", "type": "object", "anchor": "networkPolicy"}
+  networkPolicy:
+    # @schema {"name": "defaults.networkPolicy.enabled", "type": "boolean"}
+    # defaults.networkPolicy.enabled -- if network policy enabled
+    # TODO: Change this to true after implementing user custom network policy parser
+    enabled: false
```

Comment:
How can we modify this if a specific component wants to turn off NetworkPolicy?

Reply:
I think it's not that easy to do such a modification.

While it is possible to implement such a feature, the implementation may become complex depending on the use case. For example, even if you open the egress of one component, it will not be meaningful if the corresponding ingress of the component is not open. Managing all of these ingress/egress relationships is not practical, I believe.

What kind of use case are you considering?

Reply:
I fully understand the difficulty of implementing this.

However, testing to open or shutdown only some LB GW or Agents when deploying in ChaosNetwork or other special environments could be a potential usecase.

Please let me discuss whether or not to support this in this PR, since the difficulty and scope of implementation will be considerably expanded if implemented.

Reply:
We discussed offline and concluded that having an option of entirely enable/disable is enough for the use case. Thus no change is required for this PR.